.eslintrc.json

```diff
@@ -2,14 +2,14 @@
     "env": {
         "browser": true,
         "commonjs": true,
-        "es2021": true,
+        "es2015": true,
```

Comment:
Cranking down our ES version to catch any advanced feature usage at the parser-level. We can turn this up if we end up using features defined outside of `es2015`.